R/Utilities.R

```diff
@@ -468,13 +612,16 @@ OrderTADACols <- function(.data){
            "WQXConversionFactor.ActivityTopDepthHeightMeasure",
            "WQXConversionFactor.ActivityBottomDepthHeightMeasure",
            "WQXConversionFactor.ResultDepthHeightMeasure",
-           "TADA.PotentialDupRowIDs.Flag"
+           "TADA.PotentialDupRowIDs.Flag",
+           "TADA.Remove",
+           "TADA.RemoveReason",
```

Comment:
Are we using these in on TADA package side now?

Reply:
Not yet, but based on our discussion yesterday 5/31, we might want to flag the results of autoclean in TADA.Remove so users can easily know which results will not be compatible with subsequent TADA functions.